12-kubernetes-cluster.tf

```diff
@@ -129,6 +129,17 @@ resource "azurerm_kubernetes_cluster" "kubernetes_cluster" {
 
   automatic_channel_upgrade = var.enable_automatic_channel_upgrade_patch == true ? "patch" : null
   node_os_channel_upgrade   = var.enable_node_os_channel_upgrade_nodeimage == true ? "NodeImage" : null
+
+  dynamic "maintenance_window_node_os" {
+    for_each = var.enable_node_os_channel_upgrade_nodeimage != false ? [1] : [0]
+    content {
+      duration   = 4
+      frequency  = "Daily"
+      interval   = 1
+      start_time = "16:00"
+      utc_offset = "+00:00"
```

Comment:
Please don't hardcode values this way in a module, it means all envs in the future would need this exact setup if they toggle `enable_node_os_channel_upgrade_nodeimage`

Can you change these to be variables please?